### PR TITLE
fix(umip-28) Clarify that stablespread should be denominated in ethusd 

### PR DESCRIPTION
**Pull request by chicfilabae**
As discussed with Matt and Nick offline.

### CODE REVIEW

File: UMIPs/umip-28.md [hunk 1]
```diff
@@ -52,7 +52,8 @@ The value of STABLESPREAD for a given timestamp can be determined with the follo
 4. Perform A - MUSD + 1
 5. Perform the maximum of the result of step 4 and 0. This is to ensure non-negativity. 
 6. Perform the minimum of the result of step 5 and 2. This is to ensure symmetry.
-7. This result should be rounded to eight decimal places.
+7. Take the result of step 6 and divide by the value of ETHUSD
```
Comment (nicholaspai) [inline UMIPs/umip-28.md:55]: You should also specify in the `Technical Specification` section how ETHUSD will be retrieved. You could just reference the ETHUSD price identifier from another UMIP or just define 2 exchanges to use as data sources.

**Review by nicholaspai — COMMENTED** [review]
LGTM, but should we also modify the description to make it clear that we are computing stablespread / eth? (i.e. stablespread, denominated in ETH)

**Comment by smb2796** [comment]
What was the reasoning for removing Okcoin?

**Comment by chicfilabae** [comment]
@nicholaspai requested it, as "OKCoin exchange is not available on cryptowatch, I think its OK just to use bittrex for now since thats where 90% of volume is....Feel free to quote me on if you want to make that change"

I agree with the reasoning as well, so I removed it

File: UMIPs/umip-28.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The definition of this identifier should be:
  
 - Identifier name: STABLESPREAD
 - Base Currency: STABLESPREAD
```
Comment (smb2796) [inline UMIPs/umip-28.md:29]: Similar to my comments on your other UMIPs, since this is quoted in ETH I think we should list a quote currency of ETH.

Comment (chicfilabae) [reply UMIPs/umip-28.md:29]: Ack.

**Review by claytonroche — DISMISSED** [review]

File: UMIPs/umip-28.md [hunk 3]
```diff
@@ -52,7 +53,8 @@ The value of STABLESPREAD for a given timestamp can be determined with the follo
 4. Perform A - MUSD + 1
 5. Perform the maximum of the result of step 4 and 0. This is to ensure non-negativity. 
 6. Perform the minimum of the result of step 5 and 2. This is to ensure symmetry.
-7. This result should be rounded to eight decimal places.
+7. Take the result of step 6 and divide by the value of ETH/USD
```
Comment (chrismaree) [inline UMIPs/umip-28.md:56]:
```suggestion
7. Take the result of step 6 and divide by the value of ETH/USD.
```

**Review by smb2796 — APPROVED** [review]
Looks good!

**Pull request merged**